File: BACKEND/Service/Models/WorkExperienceModel.cs
namespace Service.Models
{
    public partial class WorkExperienceModel
    {
        public Guid WorkExperienceId { get; set; }
        public string? JobTitle { get; set; }
        public string? Company { get; set; }
        public DateTime? From { get; set; }
        public DateTime To { get; set; }
        public string? Description { get; set; }
        public string? Project { get; set; }
        public Guid CandidateId { get; set; }
        public virtual CandidateModel? Candidate { get; set; }
    }
}

File: FRONTEND/src/app/data/work-experience/work-experience.model.ts
import { Candidate } from "../candidate/candidate.model";

export class WorkExperience {
	workExperienceId?: string;
	jobTitle?: string;
	company?: string;
	from?: Date;
	to?: Date;
	description?: string;
	project?: string;
	candidateId?: string;
	candidate?: Candidate
}
